### PR TITLE
Add saddu6351's file

### PR DESCRIPTION
**Pull request by saddu6351**
learning how to pull a request

### CODE REVIEW

**Comment by saddu6351** [comment]
Review  is required  in this pull request

**Comment by saddu6351** [comment]
How to assign any reviewer for pull request.

**Review by SaadAfzalAzmi — CHANGES_REQUESTED** [review]
Good And Fine working

File: _posts/0000-01-02-saddu6351.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (saddu6351) [inline _posts/0000-01-02-saddu6351.md:5]: info -- "Your name is Saad "

**Review by saddu6351 — COMMENTED** [review]
done

**Review by SaadAfzalAzmi — APPROVED** [review]

**Review by saddu6351 — COMMENTED** [review]
done and dusted #1

File: _posts/0000-01-02-saddu6351.md (hunk 1 above)
Comment (SaadAfzalAzmi) [reply _posts/0000-01-02-saddu6351.md:5]: that is a good idea to add your info

**Review by SaadAfzalAzmi — APPROVED** [review]
ok

**Pull request closed without merging**